Source/WebCore/workers/service/server/SWRegistrationDatabase.cpp

```diff
@@ -248,7 +248,11 @@ bool SWRegistrationDatabase::prepareDatabase(ShouldCreateIfNotExists shouldCreat
 
     m_database = makeUnique<SQLiteDatabase>();
     FileSystem::makeAllDirectories(m_directory);
+#if PLATFORM(MAC)
+    auto openResult  = m_database->open(databasePath, WebCore::SQLiteDatabase::OpenMode::ReadWriteCreate, WebCore::SQLiteDatabase::OpenOptions::CanSuspendWhileLocked);
```

Comment:
I don't believe we need the `WebCore::` prefix on this line (used twice).

Reply:
ah right; will remove